app/src/main/res/layout/fragment_search.xml

```diff
@@ -15,181 +15,195 @@
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
+        android:layout_width="600dp"
+        android:layout_height="match_parent">
 
-        <com.google.android.material.appbar.CollapsingToolbarLayout
-            android:id="@+id/collapsingToolbarLayout"
+        <com.google.android.material.appbar.AppBarLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:background="?android:windowBackground"
-            app:layout_scrollFlags="scroll|exitUntilCollapsed|snap"
-            app:title="@string/bugTrackerTitle">
+            android:layout_height="wrap_content">
 
-            <com.google.android.material.appbar.MaterialToolbar
-                android:id="@+id/toolbar"
+            <com.google.android.material.appbar.CollapsingToolbarLayout
+                android:id="@+id/collapsingToolbarLayout"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                app:contentInsetEnd="@dimen/marginStandardMedium"
-                app:layout_collapseMode="pin">
+                android:layout_height="match_parent"
+                android:background="?android:windowBackground"
+                app:layout_scrollFlags="scroll|exitUntilCollapsed|snap"
+                app:title="@string/bugTrackerTitle">
+
+                <com.google.android.material.appbar.MaterialToolbar
+                    android:id="@+id/toolbar"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:contentInsetEnd="@dimen/marginStandardMedium"
+                    app:layout_collapseMode="pin">
 
-                <include
-                    android:id="@+id/searchBar"
-                    layout="@layout/item_search_view" />
+                    <include
+                        android:id="@+id/searchBar"
+                        layout="@layout/item_search_view" />
 
-            </com.google.android.material.appbar.MaterialToolbar>
-        </com.google.android.material.appbar.CollapsingToolbarLayout>
+                </com.google.android.material.appbar.MaterialToolbar>
+            </com.google.android.material.appbar.CollapsingToolbarLayout>
 
-        <HorizontalScrollView
-            android:id="@+id/horizontalScrollViewDummyId"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="@color/backgroundColor"
-            android:clipToPadding="false"
-            android:paddingHorizontal="@dimen/marginStandardMedium"
-            android:scrollbars="none">
-
-            <LinearLayout
-                android:layout_width="wrap_content"
+            <HorizontalScrollView
+                android:id="@+id/horizontalScrollViewDummyId"
```

Comment:
Is that id necessary ?

Reply:
No clue, you should see with @LunarX , he added it.

Reply:
Yeah, by giving it an id, the scroll view will retain how much it has scrolled when the activity is recreated. Without the idea it's reset to 0 every time. We could add a comment nontheless

Reply:
Done.